www/pages/guides/now.md

```diff
@@ -0,0 +1,27 @@
+---
+title: 'Deploy on Now'
+menu: side
+linkheadings: 3
+index: 5
+---
+
+## Deploying your site on Now
+
+Requires an account with [Vercel](https://vercel.com/) liked to your GitHub account. If you need to sign up, the process is simple as is the deployment.
+
+After you have created your Greenwood project and pushed your code to a GitHub repo...
+
+- log in to your Vercel account and proceed to the dashboard.
```

Comment:
Not to nitpick, but maybe we could have consistent capitalization for the start of each list item?